pandas/tests/io/test_parquet.py

```diff
@@ -1131,6 +1131,21 @@ def test_infer_string_large_string_type(self, tmp_path, pa):
     #     assert result["strings"].dtype == "string"
     # FIXME: don't leave commented-out
 
+    def test_non_nanosecond_timestamps(self, temp_file):
+        # GH#49236
+        import pyarrow as pa
+        import pyarrow.parquet as pq
```

Comment:
```suggestion
        pa = pytest.importorskip("pyarrow")
        pq = pytest.importorskip("pyarrow.parquet")

```

Reply:
@EduardAkhmetshin the difference is that other tests that do those imports use the `pa` fixture, which will skip the test if pyarrow is not installed. In this case we aren't using that same fixture, so this test will run and fail if pyarrow isn't present

I do agree that's a confusing way for these tests to be structured, but cleaning that up can be done in another PR

Reply:
@WillAyd I see, thanks! What is the preferred option - `importskip` or `pa` fixture? Other tests in this file use `pa` fixture.

Reply:
I would just go with @mroeschke suggestion for now